lib/ts/recipe/passwordless/types.ts

```diff
@@ -285,6 +270,10 @@ export type APIInterface = {
               preAuthSessionId: string;
               flowType: "USER_INPUT_CODE" | "MAGIC_LINK" | "USER_INPUT_CODE_AND_MAGIC_LINK";
           }
+        | {
+              status: "SIGN_IN_UP_NOT_ALLOWED";
+              reason: string;
+          }
         | GeneralErrorResponse
     >;
```

Comment:
VS code is giving me the option for user.phoneNumber in it where response is the result of calling consumeCode. I think that’s an issue somewhere in the SDK.
![Screenshot 2023-08-25 at 18 48 33](https://github.com/supertokens/supertokens-node/assets/2976287/83f814e2-5a9d-4cc2-a478-66c32e8d0dbc)